### PR TITLE
fix(FileMaker Node): Improve returned error responses

### LINKED ISSUE #6584: FileMaker Node : incorrect error handling
**Describe the bug**
When the API returns an error with status code 4XX, the node does not throw an error nor return the corresponding error response from the API.
This error occur since the API was updated and does not return a 200 status code when an error occur. It now instead returns a 4XX status code. 

**To Reproduce**
Run a create or update record action with for instance an invalid value for a fields which will trigger a validation error

**Expected behavior**
The node should return the API's error code and message in json format

**Environment (please complete the following information):**

- OS: docker 0.220.1  image

**Additional context**
I'm posting this bug report just because I'm currently creating a pull request to fix it ;)

### PR DESCRIPTION
**Pull request by airmoi**
fix #6584

### CODE REVIEW

File: packages/nodes-base/nodes/FileMaker/FileMaker.node.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"node": "n8n-nodes-base.filemaker",
-	"nodeVersion": "1.0",
+	"nodeVersion": "1.1",
```
Comment (Joffcom) [inline packages/nodes-base/nodes/FileMaker/FileMaker.node.json:3]: This is not needed and can continue to be 1.0

**Comment by Joffcom** [comment]
Hey @airmoi,

How much testing have you done with this change?

**Comment by airmoi** [comment]
I tested it against a bunch of flows using this node in production as well as test cases (flows) I made after discovering this issue.

**Comment by airmoi** [comment]
I take advantage of this pull request to update the icon

**Comment by Joffcom** [comment]
Hey @airmoi,

Did you try older versions of Filemaker as well?

**Comment by airmoi** [comment]
Yes.
This fix does not modify the legacy behavior, it only enhance support on request error

**Comment by Joffcom** [comment]
Hey @airmoi,

Perfect, In that case I think we can merge this one.

**Review by Joffcom — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@0.236.3`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.236.3)

**Pull request merged**